packages/proto-signing/src/wallet.ts

```diff
@@ -28,7 +28,13 @@ export async function executeKdf(password: string, configuration: KdfConfigurati
     case "argon2id": {
       const options = configuration.params;
       if (!isArgon2idOptions(options)) throw new Error("Invalid format of argon2id params");
-      return Argon2id.execute(password, cosmjsSalt, options);
+
+      // Emulate a slower implementation. The fast WASM code may get removed.
+      // This approximates the speed of using a pure JS implementation (@noble/hashes) in Node 22.
+      const screamTest = new Promise((resolve) => setTimeout(resolve, options.opsLimit * 250));
```

Comment:
💅 we can use `sleep` from `@cosmjs/utils` which does exactly that but is a bit more readable